client/main.go

```diff
@@ -0,0 +1,23 @@
+package client
+
+import "host/types"
+
+type Client struct {
```

Comment:
this is kind of different of what I thought.
i wish for an http client. think of this like and sdk for using your API.
it knows the route and the structs allowing for easier communication and request between services